sphinx/themes/basic/static/searchtools.js

```diff
@@ -336,6 +337,7 @@ const Search = {
             null,
             score,
             filenames[file],
+            "title",
```

Comment:
Perhaps these constant labels could be declared on a JavaScript class, similar to the way we declare some constants on the `Scorer` class in the same `searchtools.js` file?